### ISSUE TITLE
Using 'Serial1' doesn't work for F401

### ISSUE THREAD

**Issue by Hyratel** [Levi--G/USBLibrarySTM32#1]
editing to just 'Serial' works fine, suggest some IFDEF gates for F103 vs F401

**Comment by Levi--G** [comment]
Hi, Using USBSerial (CDC) doesn't work yet with the lib as it needs bidirectional support, im working on it ;)

**Comment by Hyratel** [comment]
yeah and it fails to compile because Serial1 doesn't exist on the F401 apparently. a quick edit to 'Serial' and it works fine

Edit to clarify: 'Serial1' shows up somewhere in the modified Libraries

**Comment by Levi--G** [comment]
This should be fixed in 1.0.0 release, if you still have issues you can reopen the issue.
